src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,122 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import jdk.nashorn.internal.objects.annotations.Constructor;
+
+@Entity
+public class Book {
+
```

Comment:
Normally it's good that each class has an id, so that the database doesn't use the one that come by default.